prototype/src/main/webapp/meeting-history.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>GoMeet Meeting History</title>
+  <script src="script.js"></script>
+  <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h1 style="text-align:center;">GoMeet Meeting History</h1>
+    <ul>
+        <h3><a href="view-meeting.html">Team Lunch</a></h3>
```

Comment:
Nit: 2 spaces for indentation, but should be fine since this is just a prototype 👍